src/OpenTK.Windowing/GraphicsLibraryFramework/Enums/InputMode.cs

```diff
@@ -0,0 +1,17 @@
+﻿//
+// InputMode.cs
+//
+// Copyright (C) 2018 OpenTK
+//
+// This software may be modified and distributed under the terms
+// of the MIT license. See the LICENSE file for details.
+
+namespace OpenToolkit.Windowing.GraphicsLibraryFramework.Enums
```

Comment:
Also, generally the namespaces aren't exact to the folder structure when it comes to bindings. Take a look at OpenAL, both the enums and interfaces are under `OpenToolkit.OpenAL`, despite them being in separate folders. So consider changing this to `OpenToolkit.Windowing.GraphicsLibraryFramework`, you won't be breaking the style guide as long as you don't go mad and make subsubfolders.